Tests/ApolloServerIntegrationTests/StarWarsSubscriptionTests.swift

```diff
@@ -22,7 +21,7 @@ class StarWarsSubscriptionTests: XCTestCase {
     connectionStartedExpectation = self.expectation(description: "Web socket connected")
 
     webSocketTransport = WebSocketTransport(
-      websocket: DefaultWebSocket(
+      websocket: WebSocket(
```

Comment:
Do we not still want to abstract this out for when we offer `URLSessionWebSocket` compatibility in the future?

Reply:
It is still injected, you just need something that conforms to the `WebSocketClient` protocol. Just changed the name of the "Default" one to remove "Default". 

I decided to hold off on moving the non-abstract parts of this into `Apollo` target until we do the networking layer refactor, because the API is going to likely change so drastically at that point anyways.

Reply:
OK sounds like a plan